modules/ml/src/main/java/org/apache/ignite/ml/knn/NNClassificationModel.java

```diff
@@ -50,10 +49,11 @@
     protected DistanceMeasure distanceMeasure = new EuclideanDistance();
 
     /** kNN strategy. */
-    protected NNStrategy stgy = NNStrategy.SIMPLE;
```

Comment:
I suggest don't touch this ENUm due to there is a lot of different strategies to handle weights (not only two and it can be extend in future)

Reply:
Could you please provide an example of other strategy?